packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -1114,6 +1114,58 @@ void main() {
       },
     );
 
+    testWidgets(
```

Comment:
Move this method further up. The file has test groups and this is in the state machine test group. It should be in the ui test group above.